doc/src/site/sphinx/deployment/load_mojo.rst

```diff
@@ -298,3 +300,12 @@ If cross validation was used, ie, ``setNfolds`` was called and value was higher
 metrics. If cross validation was not used, but validation frame was used, the method returns validation metrics. Validation
 frame is used if ``setSplitRatio`` was called with value lower than one. If neither cross validation or validation frame
 was used, this method returns the training metrics.
+
+Obtaining Leaf Node Assignments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To obtain the leaf node assignments, please first make sure to set ``leafNodeAssignmentsEnabled`` and
+``withDetailedPredictionCol`` to true on your MOJO settings object. The leaf node assignments are now stored
+in the ``${detailedPredictionCol}.leafNodeAssignments`` column on the dataset obtained from the prediction.
```

Comment:
Does ``${...}` have any effect in rst?

Reply:
Wasn't sure, but checked and looks ok, no special meaning. Thanks!